ui/raidboss/popup-text.ts

```diff
@@ -429,29 +622,40 @@ export class PopupText {
           // Add an additional resolved regex here to save
           // time later.  This will clobber each time we
           // load this, but that's ok.
-          trigger.filename = set.filename;
+          trigger.filename = set.filename ?? 'Unknown';
+          const id = trigger.id;
 
-          if (!trigger.regex && !trigger.netRegex) {
-            console.error('Trigger ' + trigger.id + ': has no regex property specified');
+          if (!('regex' in trigger) && !('netRegex' in trigger)) {
+            console.error(`Trigger ${id}: has no regex property specified`);
```

Comment:
I think it might make sense to change this to:

```typescript
          if (!isRegexTrigger(trigger) && !isNetRegexTrigger(trigger)) {
            console.error(`Trigger ${id}: has no regex property specified`);
```

Thoughts?

Reply:
Yeah, that seems nice, since "is this one of the two types of triggers we care about" is what this is trying to check.